<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClientWidget</class>
 <widget class="QWidget" name="AuditClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditClientWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>701</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion au serveur</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Etat de la connexion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditEtat"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>49</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditPort"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButtonConnexion">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Connexion</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Num√©ro de port du serveur :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditAdresseIP"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Adresse IP du serveur :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonQuitter">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxInformationClient">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>701</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Information Client</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>651</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOrdinateur">
       <property name="text">
        <string>Nom de l'ordinateur distant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOrdinateur"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>651</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonUtilisateur">
       <property name="text">
        <string>Nom de l'utilisateur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUtilisateur"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>651</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonArchitecture">
       <property name="text">
        <string>Architecture processeur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditArchitecture"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>130</y>
      <width>651</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButtonOS">
       <property name="text">
        <string>OS de l'ordinateur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOS"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
